src/plugins/bmqprometheus/bmqprometheus_prometheusstatconsumer.cpp

```diff
@@ -414,11 +418,18 @@ void PrometheusStatConsumer::captureQueueStats()
                 updateMetric(&confirmTimeDataPoint, labels, value);
 
                 if (role == mqbstat::QueueStatsDomain::Role::e_PRIMARY) {
-                    value = mqbstat::QueueStatsDomain::getValue(
-                        *appIdIt,
-                        d_snapshotId,
-                        mqbstat::QueueStatsDomain::Stat::e_QUEUE_TIME_MAX);
-                    updateMetric(&queueTimeDataPoint, labels, value);
+                    for (DatapointDefCIter dpIt = bdlb::ArrayUtil::begin(defs);
+                         dpIt != bdlb::ArrayUtil::end(defs);
+                         ++dpIt) {
+                        const bsls::Types::Int64 value =
```

Comment:
Looks a bit confusing as we already have `value` in outer sope. Let's either reassign the existing valiable or change the name of this one

Reply:
I replaced it by introducing another for-loop for conflicting `value`.